test/e2e/test_helm_install_upgrade.go

```diff
@@ -135,9 +136,10 @@ func TestHelmUpgrade(t *testing.T) {
 			},
 		},
 		{
-			name:             "upgrade from latest minor to current",
-			fromVersion:      "1.4.0", // renovate: datasource=docker packageName=kong/gateway-operator-oss
-			upgradeToCurrent: true,
+			name:                     "upgrade from latest minor to current",
+			fromVersion:              "1.4.0", // renovate: datasource=docker packageName=kong/gateway-operator-oss
+			upgradeToCurrent:         true,
+			upgradeToEffectiveSemver: "1.5.0",
```

Comment:
Could we add a comment here that this should be the next version to be released to get the charts templates rendered?

Reply:
Added comments. 👍